crawl-ref/source/godabil.cc

```diff
@@ -4259,7 +4302,7 @@ bool gozag_potion_petition()
             for (int i = 0; i < GOZAG_MAX_POTIONS; i++)
             {
                 prices[i] = 0;
-                int multiplier = 25; // arbitrary
+                int multiplier = 20 + random2(10); // arbitrary
```

Comment:
With the change to reduce prices of less-useful potions, this maybe should increase.